ansible/roles/schulcloud-server-init/templates/job_init.yml.j2

```diff
@@ -26,6 +26,13 @@ spec:
           subPath: update.sh
         command: ['/bin/sh','-c']
         args: ['cp /update.sh /update.run.sh && chmod +x /update.run.sh &&./update.run.sh']
+        resources:
+          limits:
+            cpu: "3000m"
+            memory: "1Gi"
```

Comment:
How have you get this value?

Reply:
random assumed from grafana values, please make any recommendations for other values, also for the dof deploy. I just pushed my last ideas